packages/orchestrator-ui-components/src/components/WFOFilterTabs/WFOFilterTabs.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { EuiTab, EuiTabs } from '@elastic/eui';
+import { useTranslations } from 'next-intl';
+
+export type FilterQuery<DataType> = {
+    field: keyof DataType;
+    value: string;
+};
+
+export type WFOFilterTab<TabType, DataType> = {
+    id: TabType;
+    translationKey: string;
+    alwaysOnFilters?: FilterQuery<DataType>[];
+};
+
+export type WFOFilterTabsProps<TabType, DataType> = {
+    tabs: WFOFilterTab<TabType, DataType>[];
+    selectedSubscriptionsTab: TabType;
+    translationNamespace: string;
+    onChangeSubscriptionsTab: (updatedSubscriptionsTab: TabType) => void;
+};
+
+export const WFOFilterTabs = <TabType extends string, DataType>({
+    tabs,
+    selectedSubscriptionsTab,
```

Comment:
since this is named WFOFilterTabs, does it still need `Subscriptions` specifically in the props?

Reply:
Done